models/coded.go

```diff
@@ -40,9 +40,9 @@ func (c *Coded) AddCodeIfPresent(codeElement xml.Node) {
 
 	//extract nullFlavor from attribute if it is NA
 	nullFlavorAttribute := codeElement.Attribute("nullFlavor")
-	nullFlavorValueSet := codeElement.Attribute("valueSet")
-	if nullFlavorAttribute != nil && nullFlavorValueSet != nil {
-		code = nullFlavorValueSet.String()
+	nullFlavorValueSet := codeElement.Attributes()
```

Comment:
I couldn't get codeElement.Attribute("valueSet") to work with namespace prefixes.